Code/Editor/AboutDialog.cpp

```diff
@@ -64,6 +56,11 @@ CAboutDialog::~CAboutDialog()
 {
 }
 
+void CAboutDialog::focusOutEvent(QFocusEvent* event)
+{
+    accept();
+}
+
```

Comment:
Might be able to use the `Qt::Tool` (which is just Popup + Dialog) to achieve the behavior you mentioned in the description.
https://doc.qt.io/qt-6/qt.html#WindowType-enum

Reply:
I already tried this ...